referrer-policy/no-referrer-policy/no-referrer-policy.subresource.py

```diff
@@ -0,0 +1,16 @@
+import os, json
+
+def main(request, response):
+    script_directory = os.path.dirname(os.path.abspath(__file__))
+    template_basename = "no-referrer-policy.subresource.template.html"
```

Comment:
Likewise, I think you'll want to turn this into a more generic loading mechanism rather than spawning a variety of .py files. Again, something to consider in future patches.